Sources/MongoSwift/Collection.swift

```diff
@@ -191,14 +214,19 @@ public struct FindOptions: Encodable {
 
     /// Convenience initializer allowing any/all parameters to be optional
     public init(allowPartialResults: Bool? = nil, batchSize: Int32? = nil, collation: Document? = nil,
-                comment: String? = nil, limit: Int64? = nil, max: Document? = nil, maxAwaitTimeMS: Int64? = nil,
-                maxScan: Int64? = nil, maxTimeMS: Int64? = nil, min: Document? = nil, noCursorTimeout: Bool? = nil,
-                projection: Document? = nil, readConcern: ReadConcern? = nil, returnKey: Bool? = nil,
-                showRecordId: Bool? = nil, skip: Int64? = nil, sort: Document? = nil) {
+                comment: String? = nil, cursorType: CursorType? = nil, hint: Hint? = nil, limit: Int64? = nil,
+                max: Document? = nil, maxAwaitTimeMS: Int64? = nil, maxScan: Int64? = nil, maxTimeMS: Int64? = nil,
+                min: Document? = nil, noCursorTimeout: Bool? = nil, projection: Document? = nil,
+                readConcern: ReadConcern? = nil, returnKey: Bool? = nil, showRecordId: Bool? = nil, skip: Int64? = nil,
+                sort: Document? = nil) {
         self.allowPartialResults = allowPartialResults
         self.batchSize = batchSize
         self.collation = collation
         self.comment = comment
+        self.cursorType = cursorType
```

Comment:
Would it make sense to note (either here or in the declaration above) that we're only storing this for the benefit of user debugging?

Reply:
yes, I will add it, for our own sake if no one else's